client/upe-onboarding-wizard/upe-preview-methods-selector/add-payment-methods-task.js

```diff
@@ -164,15 +186,29 @@ const AddPaymentMethodsTask = () => {
 				</p>
 				<Card className="add-payment-methods-task__payment-selector-wrapper">
 					<CardBody>
-						{ /* eslint-disable-next-line max-len */ }
-						<p className="add-payment-methods-task__payment-selector-title wcstripe-wizard-task__description-element is-headline">
-							{ __(
-								'Payments accepted at checkout',
-								'woocommerce-gateway-stripe'
-							) }
-						</p>
-
 						<LoadableSettingsSection numLines={ 20 }>
+							<HeadingWrapper>
+								{ /* eslint-disable-next-line max-len */ }
+								<p className="add-payment-methods-task__payment-selector-title wcstripe-wizard-task__description-element is-headline">
+									{ __(
+										'Payments accepted at checkout',
+										'woocommerce-gateway-stripe'
+									) }
+								</p>
+								{ ! areAllPaymentMethodsSelected && (
```

Comment:
I am not sure about the UX of the disappearing link. It's a nice touch but maybe it'd be better to disable it or not do anything at all? Might just be personal preference though.

Reply:
Yeah, I was also on the fence about it - removed in 6da4a856